subcommand/server-acl-init/command_test.go

```diff
@@ -408,6 +400,21 @@ func TestRun_DelayedServerPods(t *testing.T) {
 			},
 		})
 		require.NoError(err)
+		_, err = k8s.AppsV1().StatefulSets(ns).Create(&appv1.StatefulSet{
```

Comment:
@ishustava I'd like to factor this out into some helper functions but I'll do that after a review so it's easier to read